admin_guide/high_availability.adoc

```diff
@@ -542,6 +542,11 @@ See xref:../admin_guide/high_availability.adoc#ha-vrrp-id-offset[this discussion
 |0
 |See xref:../admin_guide/high_availability.adoc#ha-vrrp-id-offset[VRRP ID Offset] discussion for more details.
 
+|`--virtual-ip-groups`
+|`OPENSHIFT_HA_VIP_GROUPS`
+|
+|Number of groups to create for VRRP. If not set, a group is created for each virtual IP given with `--virtual-ips`.
```

Comment:
There is a limit of 254 VRRP IDs in an Ethernet broadcast domain.  By setting this you can control how many VRRP IDs get used for failover.  But if the number of groups is fewer than the number of IP addresses to fail over, a group will contain more than one IP address, and all of those addresses will move together.  

This is useful for cases where all of the machines in the cluster have access to the service locally (i.e. a service is being exposed with an external IP).  BUT it should not be used for services that are local to a particular machine, such as a router (unless that service is actually running on all of the nodes).

Reply:
Also you probably should disable health checks on the ipfailover since only one of the items in the group will be checked.  So if you are relying on that for reliability, it will not work and you should not use this capability.  It is best to disable health checks when working with groups.

Basically we would suggest that people only use this with OpenShift services (ExternalIP or NodePort) and disable health checks.

If they are using pods with HostNetworking (like the router), then they need to be cautious when using this and make sure they understand all of the limitations.